sensor_msgs/Image image
sensor_msgs/CameraInfo camInfo
tracker_visp/ReferenceBlock cTo_est
---
std_msgs/String caoFilePath
tracker_visp/ReferenceBlock initPose